how2meet/ui/frontend.py

```diff
@@ -117,24 +27,25 @@ def home():
     with frame("Home"):
         with ui.column().classes("w-full items-center"):
             with ui.row():
-                ui.button("New Event", on_click=lambda: ui.open(f"/new_event/{uuid.uuid4()}"))
+                ui.button("New Event", on_click=lambda: ui.open(f"/events/new/{uuid.uuid4()}"))
```

Comment:
I see you changed the `id` fields to `str` but we can probably enforce UUIDs as a type

Reply:
@BenMemberg is unsure about the uuid being generated by the frontend. need a better way to pass it around and save to db at the right time

Reply:
Yeah I would argue this should be generated by the backend, and the create url would simply be /events/new/ or /events/create/ or something.